tests/sentry/lang/native/test_cfi.py

```diff
@@ -287,5 +290,6 @@ def test_cfi_reprocessing(self, mock_cache_get, mock_attachment_get):
 
         cache_key = 'e:9dac1e3a7ea043818ba6f0685e258c09:%s' % self.project.id
         mock_attachment_get.assert_called_once_with(cache_key)
+        mock_cache_set.assert_called_with('st:b4eeed5c7008d0003cc5549c36dba6b7', CFI_CACHE)
```

Comment:
This is the important test. It checks that the CFI info inserted into the cache actually matches what we should extract from the minidump.